README.md

```diff
@@ -119,7 +120,8 @@ Want to get up and running quickly? Follow these steps:
   git clone https://github.com/documenso/documenso
   ```
 
-- Set up your `.env` file using the recommendations in the `.env.example` file.
+- Run `cp .env.example .env` to create a copy of .env file.
```

Comment:
It's a good idea to keep the original `.env.example` for backup purposes. Incase something goes wrong, the original `.env.example` will be helpful.